fwb/middleware.ts

```diff
@@ -4,7 +4,7 @@ import { authMiddleware } from "@clerk/nextjs";
 // Please edit this to allow other routes to be public as needed.
 // See https://clerk.com/docs/references/nextjs/auth-middleware for more information about configuring your Middleware
 export default authMiddleware({
-      publicRoutes: ["/"]
+      publicRoutes: ["/","/forgotpassword",'/success',]
```

Comment:
We need to discuss which routes should be public and which should not. I had trouble accessing the sign-in and sign-up page. Clerk would auto redirect me to signing in using their own sign in component.